### PR TITLE
Update bench tps default configuration

### PR DESCRIPTION
**Pull request by sagar-solana**
#### Problem

- Bench tps defaults to incredibly high tx counts that effectively makes it impossible for the cluster to progress. 
- Users are forced to pass in `--thread-batch-sleep-ms` to accurately control tps when using bench tps. This should be default 

#### Summary of Changes

- Reduced the default tx_count Config for bench tps
- Made the default `--thread-batch-sleep-ms` 1 second. That way the `--tx_count` argument always attempts a given tps.

### CODE REVIEW

**Comment by sagar-solana** [comment]
@mvines @sakridge any objections to this? 
I'm unsure if it impacts the local bench runs we've had people in the community perform.

**Comment by mvines** [comment]
No objections from me.

**Comment by sakridge** [comment]
> * Made the default `--thread-batch-sleep-ms` 1 second. That way the `--tx_count` argument always attempts a given tps.

This isn't accurate. There is time taken for generating/signing the transactions and sending them and potentially waiting for a new blockhash. So you will always get a lower TPS than what you set `--tx_count` to.

Also local testing with --tx_count=50k and 0 sleep is usually a fine configuration value. I would only change this for the testnet default.

File: multinode-demo/bench-tps.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ if [[ -z $1 ]]; then # default behavior
     --drone 127.0.0.1:9900 \
     --duration 90 \
     --tx_count 50000 \
+    --thread-batch-sleep-ms 1000 \
```
Comment (sakridge) [inline multinode-demo/bench-tps.sh]: why set it here again when it's the same as the default?

**Comment by sagar-solana** [comment]
@sakridge which way should I take this then ? 
Set the sleep to 500 as default (allowing 500ms to generate txs) and leave it up to the user to set sleep to 0ms for local testing ?

**Comment by sakridge** [comment]
> @sakridge which way should I take this then ?
> Set the sleep to 500 as default (allowing 500ms to generate txs) and leave it up to the user to set sleep to 0ms for local testing ?

We have different defaults for local and remote, right? Can't we use those to set 0s for local and 1s for remote?

Seems like the cleanest way is to set the default to sleep=1s in cli.rs and then pass sleep=0 in bench-tps.sh.

If you want some more work...one thing that sucks is to override those args for bench-tps.sh is that you have to give all the arguments again like --entrypoint and --drone even if you want to change one. Maybe we can switch to using default_arg there, but default_arg does not work with an `=` in the arg, like `--duration=50` it will choke on. Maybe we are ok with that...

**Comment by sagar-solana** [comment]
> Seems like the cleanest way is to set the default to sleep=1s in cli.rs and then pass sleep=0 in bench-tps.sh.

Alright, I did this for now.

**Pull request merged**